### PR TITLE
Update Pinecone starter plan limits

### PR DESCRIPTION
**Pull request by kacperlukawski**
Pinecone has changed its policy for the starter plan. They reduced the capacity to around 100k OpenAI ADA embeddings.

See: 
1. https://www.pinecone.io/blog/updated-free-plan/
2. https://docs.pinecone.io/docs/indexes#starter-plan

### CODE REVIEW

**Comment by joein** [comment]
Shouldn't we just write that they support 200k 768? (they've mentioned that it is proportional for other dimensionalities)

**Comment by kacperlukawski** [comment]
> Shouldn't we just write that they support 200k 768? (they've mentioned that it is proportional for other dimensionalities)

Yeah, we probably could. Let me rephrase it

**Review by joein — APPROVED** [review]
